### PR TITLE
Kibana

### PR DESCRIPTION
**Pull request by AWare**
Brings Kibana logging to Contributions. 

See
https://github.com/guardian/membership-frontend/pull/1696
https://github.com/guardian/subscriptions-frontend/pull/958
https://github.com/guardian/subscriptions-frontend/pull/946

cc: @johnduffell

### CODE REVIEW

**Comment by dominickendrick** [comment]
Hey Alex, Thanks for doing this. Because we experienced some payment issues recently, we are making some improvements to our QA processes. This will mean we won't approve or merge this PR until they those processes are done. This may be a few days.
@guardian/contributions

File: conf/PROD.public.conf [hunk 1]
```diff
@@ -66,3 +66,5 @@ contexts {
         }
     }
 }
+
```
Comment (johnduffell) [inline conf/PROD.public.conf:69]: is there a CODE env too? should be on there as well

**Comment by johnduffell** [comment]
don't forget to add the stream name to the S3 config (otherwise it won't log anything)
looks good from my point of view for whenever the team is ready

**Comment by AWare** [comment]
Thanks @dominickendrick, I'll also hold off the conf changes until this passes QA.

**Pull request closed without merging**